src/main/java/org/pantsbuild/jarjar/Wildcard.java

```diff
@@ -126,7 +126,7 @@ private static boolean checkIdentifierChars(String expr, String extra) {
           char c = expr.charAt(i);
           if (extra.indexOf(c) >= 0)
               continue;
-          if (!Character.isJavaIdentifierPart(c))
+          if (!(Character.isJavaIdentifierPart(c) || c == '-'))
```

Comment:
A comment here as to "why `-`" would be good... (because `-` is supported starting with 10?)

Reply:
I don't think 10 supports `-` I could be missing something though. I looked at 10's [Identifier spec](https://docs.oracle.com/javase/specs/jls/se10/html/jls-3.html#jls-3.8) and I don't see dash. Maybe this is to cover `module-info`? In that case, we could add another special case like the one for `package-info`.